pkg/storageincentives/agent.go

```diff
@@ -41,6 +40,8 @@ const (
 
 	// average tx gas used by transactions issued from agent
 	avgTxGas = 250_000
+
+	purgeStaleDataThreshold = 10
```

Comment:
why does this start at 10?

Reply:
I was anxious going with threshold value which would clear all stale data asap, (value 1) because there was issue reported that API returns some values, related to round, with off by one error. Therefore value 10 was set because it makes enough room to accommodate any potential off by one error.

In future, when we are more confident, this could be easily tweaked to clear all stale data asap.